api/descriptions/collections/post.md

```diff
@@ -23,5 +23,6 @@ $ curl 'https://<CONSOLE>/api/v<VERSION>/collections' \
 }'
 ```
 Any resource left unspecified, such as `hosts`, `functions`, or `clusters`, is assigned a wildcard by default.
+Resources that are not explicitly specified need to have a `*` wildcard applied to them in order to instantiate.
```

Comment:
The statement at line number 26 suggests some user action for adding a wildcard '*'. But, it's automatically gets added for instantiation as mentioned in line number 25. We can rephrase as "To instantiate a resource object, a wildcard '*' is assigned by default if the resource is left unspecified."

Reply:
Hi, 
   If you create a collection via the API and do not specify a wildcard, it will not be added. I can demo this is necessary. That is why I suggest we remove the line saying it is added by default, as it is not.

Reply:
Okay, I maybe wrong but during my testing when I created a collection via API, the wildcards got added as mentioned.

Reply:
I am not sure what operation you performed or what test you ran to do this, it's a specific use-case. This may be easier to sync on with a quick huddle over slack if you'd like. Here, @ssugandh , take a quick look at my repo on this JIRA: https://redlock.atlassian.net/browse/PCSUP-6947 . I have some images in there that should shed some light on this.

Reply:
I see that. The issue seems to be true. I verified as suggested in the Jira.

When I create a collection via API endpoint, the wildcards are added or atleast it seems to be added on the UI. I verified with the UI. But, when I use that same collection to create a rule, it throws that error like it's mentioned in the Jira. Unless, I pass a wildcard explicitly through either POST or PUT, it's treated as empty.

Reply:
Excellent work, @ssugandh 

@kdpanw Does this cover the full extent of the issue?